dfe-analytical-services/explore-education-statistics
# Create robot tests for EES-214

What does this PR do?
- Create tests for EES-214 

Description of task ? 

This PR tests the functionality in EES-214 as outlined below: 
When a user creates an amendment to an existing release, the original permalink (generated when the user first made the original release) should display an error alert to notify them that the data is out of date.  



How should this be manually tested?
- create publication & release 
- upload a subject 
- approve release 
- create table & generate permalink 
- return to admin and make amendment 
-  approve release for amendment 
- go back to permalink created previously and check the page for an error element
